app/models/user.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  has_many :posts
+  validates :name, :email, presence: true
+  validates :name, length: { minimum: 2 }
```

Comment:
- [ ] Comments must belong to User. Meaning user have many comments. We need to add `has_many :comments` here.